frontend/src/components/AnnouncementsList/AnnouncementsList.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import Announcement from '../Announcement/Announcement';
+import PropTypes from 'prop-types';
+
+const AnnouncementsList = ({ boardsIds, announcementsNumber, showBoardNames }) => {
+  const { data, isLoading, isError } = useQuery('boardsList', async () => {
+    return await fetch('/boards').then((response) => response.json());
+  });
+
+  const [announcements, setAnnouncemensts] = useState([]);
+  const [windowSize, setWindowSize] = useState('');
+
+  useEffect(() => {
+    const resizeHandler = () => setWindowSize(window.innerWidth);
```

Comment:
Tak jak rozmawialiśmy, zrób to w CSS.
W projekcie za style powinny odpowiadać CSS, a nie dodatkowy kod w komponencie.
Więc spróbuj tym sposobem, co Tobie wysłałem. ;)
To fajne, ale raczej stosowane na początku nauki, raczej jako dla początkujących w ramach ćwiczenia/ciekawostki.
Bardziej doświadczeni tak nie zrobią. Pamiętaj, że JS obciąża urządzenie użytkownika, CSS prawie w ogóle nie.

Nawet ta animacja z StarWors, co tobie pokazywałem kiedyś, to też w całości w CSS jest zrobiona. ;)

Reply:
Po wielu potyczkach się udało.